### DISCUSSION TITLE
Need help accessing paper!

### DISCUSSION THREAD

**Discussion by zoexiro** [daniel1noble/ecoevo_1000#5] [General]
Hi all, 

I am needing help to access this paper (https://doi.org/10.18475/cjos.v52i1.a2) my uni (UNSW) does not grant access to the journal and I couldn't find the paper on sci-hub. @daniel1noble I submitted the google form with as much info as I could get on the paper. If someone is able to send it to me (zoexiro@hotmail.com) I will resubmit the form with the additional info.

Cheers
Zoe Xirocostas

**Comment by elviradbastiani** [comment]
Hi Zoe,

I tried, but I couldn't. Look at the error:

[image: image.png]

Best,
Elvira

On Sat, Oct 21, 2023 at 10:55 PM Zoe Xirocostas ***@***.***>
wrote:

> Hi all,
>
> I am needing help to access this paper (
> https://doi.org/10.18475/cjos.v52i1.a2) my uni (UNSW) does not grant
> access to the journal and I couldn't find the paper on sci-hub.
> @daniel1noble <https://github.com/daniel1noble> I submitted the google
> form with as much info as I could get on the paper. If someone is able to
> send it to me ***@***.***) I will resubmit the form with the
> additional info.
>
> Cheers
> Zoe Xirocostas
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/daniel1noble/ecoevo_1000/discussions/5>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADSIKOOD2XKYCZVLUKZ7SX3YASYOBAVCNFSM6AAAAAA6KVEQEWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVG43DCMZZGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Elvira D'Bastiani (she/her/hers)
Postdoctoral researcher - Pinter-Wollman Lab
Department of Ecology and Evolutionary Biology
University of California, Los Angeles
Lab: Pinter-Wollman Lab <https://pinterwollmanlab.eeb.ucla.edu/>

*My main scientific research**: *
Article: Metz et al. 2023. Unravelling the trophic guild structure of
Neotropical Carnivora: diet specialisations, spatial variation and
phylogenetic drivers
<https://onlinelibrary.wiley.com/doi/full/10.1111/mam.12326>.  Mammal review
Article: D'Bastiani et al. 2023. Effect of host-switching on the ecological
and evolutionary patterns of parasites
<https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syad022/7142547?login=true>.
Systematic biology
Natural history Note: D'Bastiani et al. 2023. *The risks of standing out to
mate: Predation of Hylodes cardosoi by Bothrops jararaca.
<https://onlinelibrary.wiley.com/doi/full/10.1111/aec.13363>* Austral
ecology
Article: D'Bastiani et al. 2021. Disentangling the beta-diversity in anuran
parasite communities.
<https://www.cambridge.org/core/journals/parasitology/article/abs/disentangling-the-betadiversity-in-anuran-parasite-communities/163A73C786F85F9CC57C7310E7FC6D3A>
  Parasitology
Article: D'Bastiani et al. 2020. The role of ecological opportunity in
shaping host-parasite networks.
<https://www.cambridge.org/core/journals/parasitology/article/role-of-ecological-opportunity-in-shaping-hostparasite-networks/246A51E964B071322271203766A37CFF>
 Parasitology
Article: D'Bastiani et al. 2020. How deadly sarcophagid fly larvae are for
anurans? New interactions and review to Neotropical region.
<https://link.springer.com/article/10.1007/s00436-020-06613-7> Parasitology
Research

*Scientific outreach*
*Project:* *Mulheres na Ecologia
<https://mulheresecologas.wixsite.com/mulheresnaecologia/nossa-hist%C3%B3ria>*

**Reply by zoexiro** [reply]
Thank you for trying Elvira! Hopefully someone has access  and can send a copy :)

**Comment by daniel1noble** [comment]
Sorry! No luck on my end either. I'll see if I can have a crack tomorrow in the office, but it may not be accessible!

**Reply by zoexiro** [reply]
Thanks Daniel, If it's not accessible I will just make a note in the master sheet comments and move on to extracting my other papers

**Comment by daniel1noble** [comment]
Hey, sorry. Thanks for reminding me. I checked at work and I managed to download it. Sent to you via email
